app/views/buildings/_categories.html.erb

```diff
@@ -11,7 +11,8 @@
     <div class="index-items">
       <div>
         <ul class="list-unstyled">
-          <% cat.items.each do |c| %>
+          <% cat.items.sort_by{|e| e.label}.each do |c| %>
+          <% unless c.class.to_s == "Category" %>
```

Comment:
Once #505 is merged, you can use the `items(exclude: [:category])` syntax

Reply:
Should this be resolved in the new ticket for the re-use of code (shared partial/concern...)

Reply:
No, I feel like this would benefit from rebasing this branch against master  and then applying this change, and then force pushing

Reply:
Done.